### PR TITLE
fix some bug, add toast

### PR DESCRIPTION
**Pull request by anhtu268**
==> Title: fix some bug, add toast

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/14311

==> Attachment File:

### CODE REVIEW

**Comment by anhtu268** [comment]
ready

File: app/views/layouts/_alert.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<% unless flash.empty? %>
+  <script type="text/javascript">
+    <% flash.each do |f| %>
+      <% type = f[0].to_s.gsub('alert', 'error').gsub('notice', 'info') %>
```
Comment (huuphuc93) [inline app/views/layouts/_alert.html.erb]: dùng " thay cho '

File: app/views/layouts/devise.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <%= render "layouts/meta" %>
+    <title>CapitalizeManagement</title>
+    <%= stylesheet_pack_tag 'application' %>
```
Comment (huuphuc93) [inline app/views/layouts/devise.html.erb]: dùng " thay cho '

**Comment by anhtu268** [comment]
ready

**Pull request merged**